src/sidebars/star/star.less

```diff
@@ -15,34 +19,40 @@
   outline: 0;
   display: block;
   .confirmation-modal {
+    &.flipped{
+      box-shadow: -2px -2px 2px 0px @dark-background;
+    }
+    box-shadow: -2px 2px 2px 0px @dark-background;
```

Comment:
And it also takes us closer towards the goal of only using mixins for styling (#191)